packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/inverseNodeSize.st

```diff
@@ -1,4 +1,5 @@
 accessing
 inverseNodeSize
+	"Returns the factor by which the maximum capacity of a Node is scaled down"
```

Comment:
As node isn't a class we should write stuff like that with lower case letters. This should be corrected sometime.